### PR TITLE
Setup user journey

### PR DESCRIPTION
**Pull request by Valioesi**
Incorporated the user journey page and component and adapted the button loading state.
What is still missing is the /me page, which is used as link in the final modal. But that's a separate task, so I created a task in monday.

### CODE REVIEW

**Review by schaerta — APPROVED** [review]
Everything seems to be working!

File: components/Section/SectionWrapper.tsx [hunk 1]
```diff
@@ -23,15 +24,17 @@ export const SectionWrapper = ({
...
-        <h2
-          className={cN(
-            'mb-4',
-            'px-4',
-            `${colorScheme === 'colorSchemeWhite' ? 'text-violet' : ''}`
-          )}
-        >
-          {title}
-        </h2>
+        {title && (
```
Comment (nmauersberg) [inline components/Section/SectionWrapper.tsx:27]: Ah yes, this is a good catch! Was wondering about the extra spacing I saw sometimes.

**Review by nmauersberg — APPROVED** [review]
Looks good, thank you! :)

File: components/SignatureListJourney/index.tsx [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+  const { listFlow } = customUserData;
+
+  // Process url params to update user automatically
+  useEffect(() => {
+    const { step, success } = querystring.parse(window.location.search);
+
+    const params: UrlParams = {};
+
+    if (step) {
+      params.step = step as string;
```
[the remark this answers is not in the document]
Comment (Valioesi) [reply components/SignatureListJourney/index.tsx:58]: Yeah, I was thinking that we were just never going to pass an array. But I am going to check first if it is a string as you suggested!

File: context/Authentication/index.tsx [hunk 3]
```diff
@@ -57,6 +56,14 @@ export type CustomNewsletterConsent = {
   value: boolean;
 };
 
+export type ListFlow = {
```
[the remark this answers is not in the document]
Comment (Valioesi) [reply context/Authentication/index.tsx:59]: Only one list flow per user, for now. (mostly due to missing time before democracy campaign) 
Once the user has finished a step it cannot be overwritten.
Maybe in the future we need to account for multiple campaigns (round 1, round 2 or berlin, hamburg, etc), either by having an array or an object, which maps one campaign to one ListFlow object.

**Comment by Valioesi** [comment]
I am going to add one change as discussed above.

**Comment by Valioesi** [comment]
Der redirect, wenn nicht eingeloggt, funktioniert noch nicht. Aber das liegt darin, dass das Handling vom nextPage param auf der Login Seite noch nicht übertragen wurde. Kann dann im Login Ticket behandelt werden.

**Pull request merged**